src/script/cache/CacheRepository.ts

```diff
@@ -26,7 +26,7 @@ import {StorageKey} from '../storage/StorageKey';
 export class CacheRepository {
   public static CACHE_KEY = {
     // Redefine name from "sw.js"
```

Comment:
Is this comment still valid?

Reply:
Yes, still valid. But actually the wrong name for the cache, nice find!